LEGO1/beachhouseentity.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BEACHHOUSEENTITY_H
+#define BEACHHOUSEENTITY_H
+
+#include "buildingentity.h"
+
+#ifndef undefined4
+#define undefined4 int
+#endif
+
+class BeachHouseEntity : public BuildingEntity
+{
+  virtual undefined4 VTable0x50(); // vtable+0x50

```

Comment:
Is there a purpose to declaring this function and defining a type (`undefined4`) for it?